### PR TITLE
[T4A6][F09-B1] Phua Kian Ming

### PR DESCRIPTION
**Pull request by daredevil999**
Added jUnit testing for Utils Class

### CODE REVIEW

**Comment by daredevil999** [comment]
Ready for review

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Address;
+import seedu.addressbook.data.person.Email;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.Phone;
+import seedu.addressbook.data.tag.Tag;
+import seedu.addressbook.data.tag.UniqueTagList;
+import seedu.addressbook.parser.Parser;
+
```
Comment (jeffryhartanto) [inline test/java/seedu/addressbook/common/UtilsTest.java:21]: You forgot to put the header comment.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        } catch (IllegalValueException e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
+        }
+    }
+    
+    @Test
+    public void isAnyNull_singlePerson_shouldReturnTrue() {
+        try {
+            assertEquals(false, Utils.isAnyNull(new Person(new Name("person"), new Phone("12345", true),
```
Comment (jeffryhartanto) [inline test/java/seedu/addressbook/common/UtilsTest.java:42]: Better to use 'assertFalse' then you can pass the message as the first parameter.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        }
+    }
+    
+    @Test
+    public void checkNull_shouldReturnTrue() {
+        assertEquals(true, Utils.isAnyNull("1", "2", null));
+    }
+    
+    @Test
+    public void checkUniqueElements_twoIdenticalPersons_shouldReturnFalse() {
```
Comment (jeffryhartanto) [inline test/java/seedu/addressbook/common/UtilsTest.java:56]: I don't think you need to put the word 'should' in the method name.

**Comment by jeffryhartanto** [comment]
Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**